.github/workflows/test-code.yml

```diff
@@ -33,6 +33,7 @@ jobs:
         graphqlversion:
           - 5.0.0
           - 5.2.0
+          - 5.3.1
```

Comment:
Does this have any effect if the project requires the latest version anyway? Might as well remove the older versions here if you’re going to bump it in GraphQL.Relay

Reply:
I fixed GraphQL.Relay project to use $(GraphQLTestVersion) variable and verified version of output file.